cmd/storage-rest-client.go

```diff
@@ -250,7 +250,7 @@ func (client *storageRESTClient) NSScanner(ctx context.Context, cache dataUsageC
 	if err == io.EOF {
 		err = nil
 	}
-	return cache, err
+	return newCache, err
```

Comment:
Was this main issue ?

Reply:
Correct. Rest is mainly minor tweaks to make it more exact as it updates.